### DISCUSSION TITLE
createORM() can't be passed to pinia.use

### DISCUSSION THREAD

**Discussion by erikjalevik** [CodeDredd/pinia-orm#1754] [Q&A]
I wanted to give this project a spin as it looks quite neat for what we need but I'm failing at the very first hurdle.

On trying to add pinia-orm to pinia, there's immediately a TypeScript error. Here's my code:

```
import { createORM } from 'pinia-orm';
const pinia = createPinia();
pinia.use(createORM());
```

TS says:

```
Argument of type '() => { use(plugin: PiniaOrmPlugin): any; }' is not assignable to parameter of type 'PiniaPlugin'.
  Type '{ use(plugin: PiniaOrmPlugin): any; }' is not assignable to type 'void | Partial<PiniaCustomProperties<string, StateTree, _GettersTree<StateTree>, _ActionsTree> & PiniaCustomStateProperties<...>>'.ts(2345)
```

**Comment by MartinX3** [comment]
It is still an issue with 
```
    "pinia": "^2.1.7",
    "pinia-orm": "^1.7.2",
```

**Reply by CodeDredd** [reply]
Can you please make an reproduction 🙏 
I can't reproduce it for example with this: https://github.com/CodeDredd/vite-typescript-pinia-orm

**Reply by MartinX3** [reply]
Of course :)
https://github.com/MartinX3/pinia_orm_quasar_ts_error_test

I just did
1. `pnpm create quasar`
	1. With qusar cli, vite 5, typescript, scss, script setup, etc.
2. pnpm add pinia-orm
3. `.use(createORM())`
4. Updated the deps (but it happens without this step, too)
5. `quasar dev`

It works with:
```typescript
// @ts-expect-error TODO Workaround for https://github.com/CodeDredd/pinia-orm/discussions/1754 & https://github.com/CodeDredd/pinia-orm/issues/1813
  pinia.use(createORM())
```
